src/js/edu-benefits/education-wizard.js

```diff
@@ -0,0 +1,73 @@
+  function toggleClass(element, className) {
+    element.classList.toggle(className);
+  }
+  function openState(element) {
+    element.setAttribute('data-state', 'open');
+  }
+  function closeState(element) {
+    element.setAttribute('data-state', 'closed');
+    const selectionElement = element.querySelector('input:checked');
+    if (selectionElement) {
+      selectionElement.checked = false;
+    }
+  }
+  function closeStateAndCheckChild(alternateQuestion, container) {
+    const selectionElement = alternateQuestion.querySelector('input:checked');
+    if (!selectionElement) {
+      return closeState(alternateQuestion);
+    }
+    const descendentQuestion = selectionElement.dataset.nextQuestion;
+    if (!descendentQuestion) {
+      return closeState(alternateQuestion);
+    }
+    const descendentElement = container.querySelector(`[data-question=${descendentQuestion}]`);
+    closeStateAndCheckChild(descendentElement, container);
+    return closeState(alternateQuestion);
+  }
+  function reInitWidget() {
+    const container = document.querySelector('.wizard-container');
+    const button = container.querySelector('.wizard-button');
+    const content = container.querySelector('.wizard-content');
+    const radios = container.querySelectorAll('input');
+    const apply = container.querySelector('#apply-now-button');
+    const warning = container.querySelector('#transfer-warning');
+    radios.forEach(radio => {
+      radio.addEventListener('change', () => {
+        const otherChoice = radio.dataset.alternate;
+        const otherNextQuestion = container.querySelector(`#${otherChoice}`).dataset.nextQuestion;
+        if (otherNextQuestion) {
+          const otherNextQuestionElement = container.querySelector(`[data-question=${otherNextQuestion}`);
+          closeStateAndCheckChild(otherNextQuestionElement, container);
+        }
+        if (!radio.dataset.selectedForm && (apply.dataset.state === 'open')) {
+          closeState(apply);
+        }
+        if (radio.dataset.selectedForm) {
+          if (apply.dataset.state === 'closed') {
+            openState(apply);
+          }
+          if (!radio.id !== 'create-non-transfer') {
+            closeState(warning);
+          }
+          if (radio.id === 'create-non-transfer') {
+            openState(warning);
+          }
+          apply.href = `/education/apply-for-education-benefits/application/${radio.dataset.selectedForm}/introduction`;
+          return false;
```

Comment:
To satisfy: Require Consistent Returns (consistent-return)
http://eslint.org/docs/2.0.0/rules/consistent-return